Lib/importlib/abc.py

```diff
@@ -15,20 +15,36 @@
 import abc
 import warnings
 
-# for compatibility with Python 3.10
-from .resources.abc import ResourceReader, Traversable, TraversableResources
+from .resources import abc as _resources_abc
 
 
 __all__ = [
     'Loader', 'Finder', 'MetaPathFinder', 'PathEntryFinder',
     'ResourceLoader', 'InspectLoader', 'ExecutionLoader',
     'FileLoader', 'SourceLoader',
-
-    # for compatibility with Python 3.10
-    'ResourceReader', 'Traversable', 'TraversableResources',
 ]
 
 
+def __getattr__(name, canonical=_resources_abc):
+    """
+    For backwards compatibility, continue to make names
+    from canonical available through this module.
```

Comment:
Why do you need to pass `_resources_abc` into this function?  Why not just use `_resources_abc` directly?

Reply:
I wrote it this way to make the function re-usable. By passing it as a parameter, it aliases the module name, making the implementation and its docstring generic.